nrtk_explorer/library/images_manager.py

```diff
@@ -22,7 +22,13 @@ def __init__(self, server_context=None):
     def LoadImage(self, path):
         if path not in self.images:
             img = ImageModule.open(path)
-            img = img.resize((224, 224))
+            w, h = img.size
```

Comment:
Why is this needed?

Reply:
I cannot find a more general authoritative source, 224x224 is sort of a standard image size for common Models such as resnet50 https://keras.io/api/applications/resnet/

Reply:
Actually the original Dataset size is not that high res: look at this exploration:

```
→ find . -iname '*.tif' -exec file '{}' \; |  sed -rn 's/.*height=([0-9]+).*width=([0-9]+).*/\1x\2/p' | sort | uniq -c
     15 256x256
    859 257x257
      2 424x499
     10 480x640
     20 499x499
      2 512x512
```

Reply:
The visdrone dataset has a very different aspect and resolutions, here it is:

→ find . -iname '*.jpg' -exec file '{}' \; | grep -Po '\d+x\d+' | grep -v '1x1' | sort | uniq -c
    167 1360x765
    274 1400x1050
    933 1400x788
    151 1916x1078
     36 1920x1080
     49 960x540